docs/source/installation/for_dev.rst

```diff
@@ -0,0 +1,126 @@
+Manim Installation For Developers
+=================================
+
+	This documentation is for developers who wats to contribute to ``ManimCommunity/manim``.
+
+Installing Poetry
+*****************
+
+Poetry can be easily installed in any OS by just running the below command.
+
+.. code-block:: bash
+	
+	curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
```

Comment:
Question: Is this the only, or the official way to do it?

Reply:
It's the official way to do. Check https://python-poetry.org

Reply:
Developers can install `poetry` with `pip`. See : https://python-poetry.org/docs/#alternative-installation-methods-not-recommended

They have mentioned that it is not recommended, but purely because installing with `pip` requires the developer to install `poetry` separately for each python version they have on their computer, for each project (which they will develop on multiple python versions) that requires poetry.

I think at least a large majority, if not all developers will develop manim only on a single python version, so installing poetry via pip should pose no issues.

Reply:
So what should be done? AFAIK, the doc explains it, and also I have linked to the site you linked in the bottom. So, I think it is good. Also, we should be recommending, not recommended, ways to install.

Reply:
> So what should be done?

My suggestion would be to also include a note after the "recommended" instructions, the one which uses `curl`, saying that you can also use `pip` to install `poetry`, but have to install it for every python version you use to develop manim.

Maybe add it as a note, like the [note about `BasicTeX` vs `MacTeX` that was in the readme, and is now in the docs](https://manimce.readthedocs.io/en/latest/installation/mac.html)?

> we should be recommending, the recommended ways to install.

My point is that the reason `poetry` recommend using the script they provide is because it works across multiple python versions. Most, if not all `manim` developers will only develop `manim` on a single `Python` version, so we don't have to worry about it not being a "recommended" method.